src/index.css

```diff
@@ -1,9 +1,229 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
 
 body {
+  margin: 20px;
   text-align: center;
+  align-items: center;
+  margin-left: 12rem;
+  background-color: rgb(204, 209, 211);
+}
+
+.navigation-bar {
+  display: flex;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  background-color: #fff;
+  border-bottom: 1px solid gray;
+  margin-left: 4rem;
+  margin-top: 1.5rem;
+}
+
+.main-heading {
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+  padding-top: 1rem;
+  padding-left: 2rem;
+}
+
+.thelinks {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  padding-top: 2rem;
+}
+
+.ho a {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 900;
+  color: rgb(26, 24, 24);
+  text-decoration: none;
+  margin: 4rem;
+}
+
+.ca a {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: normal;
+  color: rgb(118 113 113);
+  text-decoration: none;
+}
+
+.form-container {
+  margin: auto;
+  padding-left: 4rem;
+  padding-right: 4rem;
+  padding-bottom: 8rem;
+  background-color: #fff;
+  width: 53rem;
+  margin-left: 4rem;
+}
+
+#category {
+  width: 18.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.813rem 0.688rem 0.3rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #888;
+  background-color: hsl(11deg 80% 38% / 8%);
+}
+
+#title {
+  width: 41.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.4rem 0.688rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #888;
+  background-color: hsl(11deg 80% 38% / 8%);
+}
+
+.addbtn {
+  width: 11.938rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+  margin-top: 1rem;
+  border-radius: 0.5rem;
+  background: #0290ff;
```

Comment:
Good job using the correct values for most of the elements of the app! Let's keep trying until we get each one of them correct! :sparkles: